### ISSUE TITLE
Audio and video options

### ISSUE THREAD

**Issue by rebeccaringuette** [nasa/Transform-to-Open-Science#654]
The material in each module is overall a great resource, but too lengthy to expect most scientists to complete. A great way to meet them where they are would be to create a short summary video (maybe 5ish minutes) on the main takeaways from each module, specifically focusing on what simple changes they can make now (and how it benefits them) to increase the openness of their efforts based on the module content. For deeper dives, the team could create short 2-3 minute videos on specific topics (e.g. What is a PiD and why should I get one?) and longer more in depth videos for tutorials (e.g. the earth science notebook tutorial example). 
Another way to decrease the demand for this on the community would be to add an audio book version of this course. I have seen this done for other courses and it was quite useful. In that case, I was able to listen to the material while I was driving, stop for a few minutes and take the quiz, and then move on to the next lesson. It would be great if such an approach was implemented for this course. That way, it would be easier for the community to fit into their schedules and still get the badges.

**Comment by g-r-a-e-m-e** [comment]
An audio/video version of the material would be great! I could absolutely see the value of a condensed "Key Points" version for more experienced people. However, I would caution against oversimplifying the material for the sake of time as it could leave students/early-career researchers without sufficient context and understanding. This comes from the perspective of a student and someone working to transition careers from data science to physical science, though, so please take my .02 with a grain of salt.

**Comment by rebeccaringuette** [comment]
For those who are already over maxed on time but want to do something, a
few short videos would be great. These could be focused on the 'good,
better, best' triage I saw in one of the early modules (can't find it now).
Maybe '5 simple things you can do now to increase your publication's
citation rate' where some recommendations from the coursework are included
after a brief summary of backing research (e.g. include citations to
datasets and software in their publications, getting an ORCiD, including
their ORCiD on their publication, including a section on how to run their
code or use the datasets, etc). Similar videos could be done on topics
important to the typical research, such as '3 things I wish I had done
earlier to make my data more open' (e.g. choose a common data format, add
variable names and units to the metadata, indicate which version of my
script made the dataset and why), '4 ways to make peer review easier'
(writing down which script and which dataset you used in the publication
and why, writing down how to run the scripts, etc), and similarly short
videos with high relevance to the general researcher. Such resources would
be very useful to the community. I am not saying that such videos should
earn badges, but that these types of videos would help the community step
into the material. A comment/link at the end of each video to the
curriculum would be great (e.g. 'to learn more...').

**Comment by nasacrawford** [comment]
Thanks for the suggestion!

Just adding a note, that I had heard feedback from someone else yesterday that an audio version would also be a way to make the course overall more accessible.

**Comment by rebeccaringuette** [comment]
Yes, exactly. Thanks!

**Comment by bressler95tops** [comment]
I also agree this is a great idea for making the curriculum more accessible.  It may be worthwhile for us to add a label for feature requests / suggestions for these types of submissions.
